docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Contract Address** - Click the address to Address Detail page.
+- **(Receiving) Address** - Click the address receiving asset to Address Detail page.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+### Pending Transactions
+Currently this feature is not enabled.
+
+### Verified Contracts 
+A verified contract shares its source code with the public, including the compiler setting. The verification process compares the generated bytecode with the one deployed on-chain.
+
+The Verified Contracts feature provides key information about the contract.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-verified-contract.png").default} alt="verified-contracts" width="900" height="700" />
+</p>
+
+
+Key information includes:
+
+- **Total Number of Contracts in the Last 24 hours** - The total number of verified contracts in the last 24 hours.
+- **Address** - The contract address. 
+- **Balance** - The total amount of tZEN associated with the smart contract address.
+- **Transaction (Txns)** - The number of transactions associated with the verified contract.
+- **Compiler** - The name of the compiler type.
+- **Version** - The version number of the compiler type.
+- **Optimization** - A green check mark indicates that optimization is applied. A red cross indicates that optimization was not applied.
+- **Construction Args** - A green check mark indicates that construction arguments were used. A red cross indicates that construction arguments were not used.
+- **Verified** - A timer that displays the elapsed time since the contract was verified.
+- **Market Cap** - The value of the token if it is bought and sold with fiat currency.
+
+# Tokens Menu
+
+The Tokens menu has two feature options:
+
+- All
+- tZEN 
+
+
+## All (Tokens)
+
+The All (Tokens) option displays all ERC-20 tokens in the blockchain. 
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-alltokens.png").default} alt="all-tokens" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Token (name)** - The name of the token.
+- **Address** - The wallet address associated with the token. Click the address to **Address Detail** page.
+- **Total Supply** - The total amount of available tokens.
+- **Holder Count** - The number of accounts with some amount of the token listed.
+
+## tZEN (Address)
+
+The tZEN (Address) option displays all addresses with tZEN balances in the blockchain. 
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-tzentokens.png").default} alt="tzen-tokens" width="500" height="300" />
+</p>
+
+Key information includes:
+
+- **Address** - The wallet address. Click the address to **Address Detail** page.
+- **Balance** - The total amount of tZEN.
+- **Txn Count** - The number of transactions associated with this tZEN that are sent.
+
+
+# API Menu

```

Comment:
Same thing is going on here as far as sections/sub-section issue

Reply:
made correction